website/build.gradle

```diff
@@ -152,36 +188,49 @@ def getBaseUrl() {
   return "${System.getProperty('user.name')}-${buildContext}"
 }
 
+def getBranchRepo() {
+  def stdout = new ByteArrayOutputStream()
+  exec {
+    commandLine 'git', 'config', '--get', 'remote.origin.url'
+    standardOutput = stdout
+  }
+  def matches = (stdout =~ /git@github\.com:(\w+\/\w+)\.git/)
```

Comment:
Would the remote be same as `project.findProperty('gitPublishRemote')`

Reply:
It works differently locally and on Jenkins. The basic idea is that we want files to point to the pull request's branch for testing and staging, but to point to the master branch once it's in production. I eventually changed all this into this:
```
/**
 * Gets the branch repository where the new files are located. This is used to point
 * all the GitHub and Colab files to the pull request's branch when testing, while
 * pointing them to the apache/beam master branch on production. This is done so tests
 * and staged versions work even when the files don't exist in master yet.
 */
def getBranchRepo() {
  // Jenkins stores the GitHub author in $ghprbPullAuthorLogin
  def author = System.env.ghprbPullAuthorLogin
  if (author == null) {
    // If the author is not defined, it's most probably running locally.
    // Try to infer the author from the remote URL
    for (remote in grgit.remote.list()) {
      if (remote.getName() == 'origin') {
        // remote.url = 'git@github.com:author/beam.git'
        author = remote.url.split(':')[1].split('/')[0]
        break
      }
    }
  }
  if (author && grgit) {
    return "${author}/beam/blob/${grgit.branch.current().getName()}"
  }
  return "apache/beam/blob/master"
}
```